backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/TaskServiceScheduler.java

```diff
@@ -110,7 +110,7 @@ public void unschedule(String name) {
 
   public void update(ScriptTask scriptTask) {
     // if no cron we remove
-    if (scriptTask.getCronExpression() == null) {
+    if (scriptTask.getCronExpression() == null || scriptTask.isDisabled()) {
```

Comment:
this line adds the missing check , the other code fixes the object loading from the db